_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,275 @@
+
+    // Deploy http bridge
+    KafkaBridgeResource.kafkaBridge(CLUSTER_NAME, KafkaResources.plainBootstrapAddress(CLUSTER_NAME), 1)
+        .editSpec()
+            .withNewConsumer()
+                .addToConfig(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
+            .endConsumer()
+        .endSpec()
+        .done();
+}
+```
+
+As you can see, this test case ensures that we are able to receive messages from the `KafkaBridge` component.
+In the setup phase we deploy `cluster-operator`, a single one node Kafka cluster and also the `KafkaBridge` itself.
+Everything is deployed into the real Kubernetes cluster.
+All test cases with related test suites can be found in the [systemtest](https://github.com/strimzi/strimzi-kafka-operator/tree/master/systemtest) 
+module.
+Also worth mentioning is that our system tests currently take around ~30 hours. 
+We have 3 separate sub-sets of tests, which run in parallel so it is "only" approximately 10h per each sub-set.
+Additionally, there is also a mvn profile for the main groups - `acceptance`, `regression`, `smoke`, `bridge` and `all`,
+but we suggest to use a profile with id `all` (default) and then include or exclude specific groups.
+If you want specify the profile, use the `-P` flag, for example `-Psmoke`. Here is an example of how we trigger some profiles:
+
+```
+Generic definition
+
+mvn -f ${workspace}/systemtest/pom.xml -P all verify -Dgroups=${testProfile} -DexcludedGroups=${excludeGroups}
+
+Acceptance profile
+
+mvn -f ${workspace}/systemtest/pom.xml -P all verify -Dgroups=acceptance -DexcludedGroups=""
+```
+
+All available test groups are listed in [Constants](https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/src/main/java/io/strimzi/systemtest/Constants.java)
+class. If you're interested, you can view our testing [document](https://github.com/strimzi/strimzi-kafka-operator/blob/master/development-docs/TESTING.md).
+
+## How system tests run in relation with Kubernetes
+
+As mentioned in the previous section, system tests run against real Kubernetes cluster.
+You can run these tests against any Kubernetes cluster type such as `Minikube`, `Minishift`, big `Kubernetes` or `Openshift`.
+In the `test` module we have our a representation of a cluster which to decide on which environment it is running.
+Moreover, we are using `Azure pipelines` to trigger these tests and using `Minikube`.
+
+## How to run them
+
+Running system tests is nothing to be scared of. In fact, it's easy.
+
+The steps are as follows:
+
+1. Fork the [strimzi-kafka-operator](https://github.com/strimzi/strimzi-kafka-operator)
+2. Build the whole project in root directory - `mvn clean install -DskipTests`
+3. Set up your Kubernetes cluster
+4. Log in to your Kubernetes cluster
+5. Try to start some of our tests inside system the `test` module, for instance - `io.strimzi.systemtest.bridge.HttpBridgeST.testSendSimpleMessage()` (this takes around 5 minutes)
```

Comment:
Give an example command for running this test, or explain that you can run it from the IDE. Also, aren't there some env vars they need to configure?

Reply:
There is no need to configure any env in the IDE only if they want to use their local images. Fortunately, I already described that in that section.